source/time_tables/departure_boards.cpp

```diff
@@ -65,6 +66,18 @@ static bool is_terminus_for_all_stop_times(const std::vector<routing::datetime_s
     return !stop_times.empty();
 }
 
+void update_display_information(bool& vj_found,
+                                const std::pair<unsigned int, const navitia::type::StopTime*>& dt_st,
+                                pbnavitia::PtDisplayInfo* pt_display_information,
+                                PbCreator& pb_creator) {
+    if (!vj_found && dt_st.second != nullptr) {
+        auto vj = dt_st.second->vehicle_journey;
```

Comment:
for better readbility... (it's just pure c++ shit)
```suggestion
        const auto* vj = dt_st.second->vehicle_journey;
```